### PR TITLE
Env is bad

### PR DESCRIPTION
**Pull request by lanza**

### CODE REVIEW

**Comment by lanza** [comment]
@compnerd

File: Makefile [hunk 1]
```diff
@@ -22,13 +22,13 @@ ifeq ($(BuildType),Debug)
...
-endif
-
-ifeq ($(BuildType),Release)
+else ifeq ($(BuildType),Release)
   CMakeBuildType := RelWithDebInfo
   AssertsEnabled := YES
   AssertsVariant := Asserts
   InstallVariant := -stripped
+else
+  $(error BuildType should be either Debug or Release)
```
Comment (compnerd) [inline Makefile]: Wasn't this part of another change?

File: cmake/modules/WindowsSDK.cmake [hunk 2]
```diff
@@ -72,9 +72,9 @@ function(windows_compute_link_flags variable)
 endfunction()
 
 function(windows_create_vfs_overlay flags)
-  get_filename_component(VCToolsInstallDir $ENV{VCToolsInstallDir} ABSOLUTE)
-  get_filename_component(UniversalCRTSdkDir $ENV{UniversalCRTSdkDir} ABSOLUTE)
-  set(UCRTVersion $ENV{UCRTVersion})
+  #get_filename_component(VCToolsInstallDir ${VCToolsInstallDir} ABSOLUTE)
+  #get_filename_component(UniversalCRTSdkDir ${UniversalCRTSdkDir} ABSOLUTE)
+  #set(UCRTVersion ${UCRTVersion})
```
Comment (compnerd) [inline cmake/modules/WindowsSDK.cmake]: Why comment it out?  Just delete it if it isn't needed.

**Review by compnerd — COMMENTED** [review]
I think that this needs to rebased.

**Pull request merged**